cypress/utils/data_utils.ts

```diff
@@ -351,3 +354,18 @@ export function getRulesData(targetData): RulesRepositoryFields | RulesManualFie
         rulesetPaths: targetData.rulesFiles,
     };
 }
+
+export function getRandomAssetGeneratorData(
+    templateRepository: GeneratorTemplateRepository,
+    generatorType?: GeneratorType
+): AssetGeneratorData {
+    if (!generatorType) {
+        generatorType = GeneratorType.Helm;
+    }
+    return {
+        name: `Generator-${getRandomNumber()}`,
+        generatorType: generatorType,
+        templateRepository: templateRepository,
+        description: getDescription(),
+    };
+}
```

Comment:
One minor comment so that generatorType defaults to Helm, otherwise the PR looks good .

```
export function getRandomAssetGeneratorData(
  templateRepository: GeneratorTemplateRepository,
  generatorType: GeneratorType = GeneratorType.Helm
): AssetGeneratorData {
  return {
    name: `Generator-${getRandomNumber()}`,
    generatorType,
    templateRepository,
    description: getDescription(),
  };
}
```